core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -2396,7 +2507,7 @@ private[spark] class DAGScheduler(
    * @param stagesToRollback stages to roll back
    * @return Shuffle map stages which need and can be rolled back
    */
-  private def abortStageWithInvalidRollBack(stagesToRollback: HashSet[Stage]): HashSet[Stage] = {
+  private def abortStagesUnableToRollback(stagesToRollback: HashSet[Stage]): HashSet[Stage] = {
```

Comment:
how about `filterAndAbortUnrollbackableStages`? It returns rollbackable stages which is like `.filter`.

Reply:
Sounds good. Updated. Thanks.